From a6c547e46bbadd2c08be9944a85308f6625263cb Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 29 Aug 2022 23:15:44 -0700
Subject: [PATCH] examples: Include alloca.h for strdupa

musl defines strdupa via a macro which uses alloca() therefore include
the header to get the prototype

Upstream-Status: Submitted [https://github.com/linux-rdma/rdma-core/pull/1212]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 libibverbs/examples/asyncwatch.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libibverbs/examples/asyncwatch.c b/libibverbs/examples/asyncwatch.c
index 724796e58..842b94180 100644
--- a/libibverbs/examples/asyncwatch.c
+++ b/libibverbs/examples/asyncwatch.c
@@ -36,6 +36,7 @@
 #include <endian.h>
 #include <getopt.h>
 #include <string.h>
+#include <alloca.h>
 
 #include <util/compiler.h>
 #include <infiniband/verbs.h>
-- 
2.37.2

